service/pkg/server/start.go

```diff
@@ -72,6 +72,19 @@ func Start(f ...StartOptions) error {
 		return fmt.Errorf("issue registering services: %w", err)
 	}
 
+	// verify dependencies
+	slog.Info("verifying dependencies")
+	for name, service := range conf.Services {
+		if service.Enabled && (service.Dependencies != nil) {
```

Comment:
@elizabethhealy I think we want to check if remote is defined here because in the near future for example kas will run a a separate deployment.

Reply:
if Remote is defined is should enabled be true? or does `enabled` pertain to whether its enabled here, not just anywhere

Reply:
enabled is only for within the current running service.